dstep/translator/IncludeHandler.d

```diff
@@ -84,58 +92,119 @@ class IncludeHandler
         ];
     }
 
+    this (Options options)
+    {
+        import std.format;
+
+        this.options = options;
+
+        if (options.packageName != "")
+        {
+            auto inputFiles = options.inputFiles.filter!(
+                x => x != options.inputFile)();
```

Comment:
I prefer to not include the last set of parentheses in cases where there are only template parameters.